GraphView/po/pl-local.po

```diff
@@ -0,0 +1,634 @@
+msgstr "Brak zakładek dla tego wykresu..."
+
+#: GraphView/graphview.py:1292
+msgid "No other bookmarks..."
+msgstr "Brak innych zakładek..."
+
+#: GraphView/graphview.py:1297 GraphView/graphview.py:1299
+#: GraphView/search_widget.py:224 GraphView/search_widget.py:243
+#, python-format
+msgid "found: %s"
+msgstr "znaleziony: %s"
+
+#: GraphView/graphview.py:1309
+#, python-format
+msgid ""
+"Add active person to bookmarks\n"
+"%s"
+msgstr ""
+"Dodaj aktywną osobę do zakładek\n"
+"%s"
+
+#: GraphView/graphview.py:1396
+#, python-format
+msgid ""
+"Person <b><i>%s</i></b> is not in the current view.\n"
+"Do you want to set it active and rebuild view?"
+msgstr ""
+"Osoba <b><i>%s</i></b> nie jest w bieżącym widoku.\n"
+"Ustawić jako osobę aktywną i odświeżyć widok?"
+
+#: GraphView/graphview.py:1399
+msgid "Change active person?"
+msgstr "Zmienić aktywną osobę?"
+
+#: GraphView/graphview.py:1400 GraphView/graphview.py:4355
+#: GraphView/graphview.py:4384
+msgid "Yes"
+msgstr "Tak"
+
+#: GraphView/graphview.py:1400 GraphView/graphview.py:4355
+#: GraphView/graphview.py:4384
+msgid "No"
+msgstr "Nie"
+
+#: GraphView/graphview.py:2459 GraphView/graphview.py:2492
+msgid "Relationship loop detected"
+msgstr "Wykryto pętlę relacji"
+
+#: GraphView/graphview.py:2532
+msgid ""
+"You try to build graph containing more then 1000 persons. Not all persons "
+"will be shown in the graph."
+msgstr ""
+"Próbujesz wyświetlić wykres zawierający więcej niż 1000 osób. Nie wszystkie osoby "
+"zostaną pokazane na wykresie".
+
+#: GraphView/graphview.py:2535
+msgid "Incomplete graph"
+msgstr "Wykres niekompletny"
+
+#: GraphView/graphview.py:2888
+msgid "Image on right side"
+msgstr "Obraz po prawej stronie"
+
+#: GraphView/graphview.py:2909
+msgid "Image on left side"
+msgstr "Obraz po lewej stronie"
+
+#: GraphView/graphview.py:2930
+msgid "Normal"
+msgstr "Normalny"
+
+#: GraphView/graphview.py:3007 GraphView/graphview.py:3008
+#: GraphView/graphview.py:3020 GraphView/graphview.py:3021
+#, python-format
+msgid "%s "
+msgstr "%s "
+
+#: GraphView/graphview.py:3238
+msgid "<b>Tags:</b>"
+msgstr "<b>Tagi:</b>"
+
+#: GraphView/graphview.py:3618
+msgid "Lines type"
+msgstr "Typ linii"
+
+#: GraphView/graphview.py:3622
+msgid "Direct"
+msgstr "Proste"
+
+#: GraphView/graphview.py:3630
+msgid "Curves"
+msgstr "Krzywe"
+
+#: GraphView/graphview.py:3638
+msgid "Ortho"
+msgstr "Ortho"
+
+#: GraphView/graphview.py:3659
+msgid "Copy"
+msgstr "Kopiuj"
+
+#: GraphView/graphview.py:3662 GraphView/graphview.py:3921
+msgid "Delete"
+msgstr "Usuń"
+
+#: GraphView/graphview.py:3668 GraphView/graphview.py:3927
+msgid "Tags"
+msgstr "Tagi"
+
+#: GraphView/graphview.py:3670
+msgid "Select tags for person"
+msgstr "Wybierz tagi dla osoby"
+
+#: GraphView/graphview.py:3673 GraphView/graphview.py:3932
+msgid "Organize Tags..."
+msgstr "Organizuj Tagi..."
+
+#: GraphView/graphview.py:3677 GraphView/graphview.py:3936
+msgid "Spouses"
+msgstr "Małżonkowie"
```

Comment:
I think it is difficult to find exact translation for this. "Spose" is broader term than "Małzonkowie". Perhaps we can leave it as it is. Not sure how it is translated in Gramps.

Reply:
I know the concept is broader in this case. However, I propose to leave "Małżonkowie".